python/hsml/model_serving.py

```diff
@@ -0,0 +1,115 @@
+#
+#   Copyright 2022 Logical Clocks AB
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+
+from typing import Union, Optional
+
+from hsml.core import serving_api
+from hsml.model import Model
+from hsml.predictor import Predictor
+from hsml.predictor_config import PredictorConfig
+from hsml.deployment import Deployment
+from hsml.transformer_config import TransformerConfig
+
+
+class ModelServing:
+    DEFAULT_VERSION = 1
+
+    def __init__(self, project_name: str, project_id: int):
+        self._project_name = project_name
+        self._project_id = project_id
+
+        self._serving_api = serving_api.ServingApi()
+
+    def get_deployment(self, id: int):
```

Comment:
Maybe nicer UX to have a get_deployment_by_id and have the get_deployment instead take as a parameter the name. I think most users will want to pass the name directly.

Reply:
Yes, good point.